site/docs/concepts/optimization-techniques/kvcache-eviction-algorithm.md

```diff
@@ -11,7 +11,7 @@ The cache eviction algorithm is designed to manage KV (Key-Value) cache memory f
 ## Conceptual Model
 The KV cache for each sequence is divided into three logical areas:
 
-![KV cache layout with cache eviction](/img/kv-cache-areas-diagram.svg)
+![KV cache layout with cache eviction](./../../../static/img/kv-cache-areas-diagram.svg)
```

Comment:
As I can see on the deployed version from master, `kv-cache-areas-diagram.svg` image is displayed correctly:
https://openvinotoolkit.github.io/openvino.genai/docs/concepts/optimization-techniques/kvcache-eviction-algorithm 
Am I missing something?

Reply:
True. This fixes repo's render: https://github.com/openvinotoolkit/openvino.genai/blob/master/site/docs/concepts/optimization-techniques/kvcache-eviction-algorithm.md

Reply:
It should work without changing image path from absolute to relative.
Docusaurus should prepend `baseUrl` (which is set in docusaurus.config.ts) to all paths under the hood. Otherwise all other image URLs would be affected.

Reply:
GitHub pages are fine. The problem is with GitHub's/VS Code markdown view